.github/workflows/linux.yml

```diff
@@ -57,6 +57,14 @@ jobs:
         cd "${{runner.workspace}}/DiligentTools/DiligentTools/Tests/DiligentToolsTest/assets"
         ${{runner.workspace}}/build/DiligentTools/Tests/DiligentToolsTest/DiligentToolsTest
 
+    - name: Upload artifact
+      uses: actions/upload-artifact@v2
+      if: ${{ success() && matrix.config == 'Release' }}
+      with:
+        name: Diligent-RenderStatePackager-Linux-x64-GCC9-${{ matrix.config }}
```

Comment:
Please rename this to DiligentTools-Linux... to be consistent with how artifacts for Core and named.
Besides RenderStatePackager, there is also HLSL2GLSLConverter in the archive.